### PR TITLE
Add trainee teacher database section

### PR DESCRIPTION
**Pull request by jesseyuen**
This PR should hopefully have all the bits and bobs to add the Register teacher trainee section but it won't show in the UI until I add a post. I'm seperating the setup PR and the first post PR for neatness sake. 

The first post and user needs will be following early next week.

### CODE REVIEW

File: app/posts/register-trainee-teachers/register-trainee-teachers.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+tags: false
+layout: collection
+title: Register trainee teachers
+description: A service for collecting and maintaing trainee teacher registration data
+# related:
+#   items:
+#   - text: Prototype
+#     href: https://manage-courses-prototype.herokuapp.com
```
Comment (paulrobertlloyd) [inline app/posts/register-trainee-teachers/register-trainee-teachers.md:9]: Does @edwardhorsford have a public link for this yet?

**Comment by paulrobertlloyd** [comment]
The BAT Design History grows another leg! 🐙

**Comment by paulrobertlloyd** [comment]
LGTM. The section isn’t showing, but I assume that’s because there are no posts yet?

File: app/posts/register-trainee-teachers/register-trainee-teachers.md (hunk 1 above)
Comment (jesseyuen) [reply app/posts/register-trainee-teachers/register-trainee-teachers.md:9]: Patience @paulrobertlloyd!

**Comment by jesseyuen** [comment]
> LGTM. The section isn’t showing, but I assume that’s because there are no posts yet?

Yep, it shows once I add the first post. @paulrobertlloyd

**Review by paulrobertlloyd — APPROVED** [review]
:shipit:

File: app/posts/register-trainee-teachers/register-trainee-teachers.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+tags: false
+layout: collection
+title: Register trainee teachers
+description: A service for collecting and maintaing trainee teacher registration data
```
Comment (fofr) [inline app/posts/register-trainee-teachers/register-trainee-teachers.md:5]: Typo, maintaining

Comment (jesseyuen) [reply app/posts/register-trainee-teachers/register-trainee-teachers.md:5]: Doh!

**Pull request merged**